app/view/media/sdl/controllsView.js

```diff
@@ -82,7 +82,21 @@ SDL.SDLMediaControlls = Em.ContainerView.create(
               'bc-item-big',
               'prevcd'
             ],
-            icon: 'images/media/ico_prew.png',
+            onIconChange: function() {
+              let icon = "images/media/ico_prew.png"; 
+              for (var i = 0; i < SDL.SDLModel.data.registeredApps.length; i++) {
+                appID = SDL.SDLModel.data.registeredApps[i].appID;
+                if (appID == SDL.SDLMediaController.currentAppId) { 
```

Comment:
```suggestion
                if (SDL.SDLMediaController.currentAppId
                  == SDL.SDLModel.data.registeredApps[i].appID) { 
```

only read once, i think we should just compare it here